http/client_test.go

```diff
@@ -244,6 +244,78 @@ func (s *HTTPClientTestSuite) TestResourceLimitExceeded(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `Resource limit exeeded at URL http://.*`)
 }
 
+func (s *HTTPClientTestSuite) TestRetryHttpDate(c *gc.C) {
```

Comment:
Can't we combine this test with the one above into common code and just pass in a string value to use for the retry after header value.

Reply:
done